### PR TITLE
Upload Omega Hover Nerf Patch

### PR DESCRIPTION
**Pull request by JotaroPowered**
Uploading this patch as a standalone.

### CODE REVIEW

File: OmegaHoverNerf.mlua [hunk 1]
```diff
@@ -0,0 +1,116 @@
+--[Patch]--
+Title("Omega Hover Nerf")
+Author("Rei-San(Reimous,GhostTH)")
+Platform("Xbox 360")
+Blurb("Limits Omega Hovering")
```
Comment (HyperBE32) [inline OmegaHoverNerf.mlua:5]: Change the Blurb field to `Limits Omega's hovering to only a few seconds to prevent exploits.`

**Review by hyperbx — CHANGES_REQUESTED** [review]

**Review by JotaroPowered — COMMENTED** [review]
Changes look good

**Comment by HyperBE32** [comment]
Those aren't changes, you need to commit them yourself.

File: Restore Multiplayer Particles.mlua [hunk 2]
```diff
@@ -0,0 +1,15 @@
+ --[Patch]--
+Title("Restore Multiplayer Particles")
```
Comment (HyperBE32) [inline Restore Multiplayer Particles.mlua:2]: Change the Title field to `Fix Multiplayer Particles`

File: Restore Multiplayer Particles.mlua [hunk 3]
```diff
@@ -0,0 +1,15 @@
+ --[Patch]--
+Title("Restore Multiplayer Particles")
+Author("Reimous")
+Platform("Xbox 360")
+Blurb("Adds Particles back to multiplayer")
```
Comment (HyperBE32) [inline Restore Multiplayer Particles.mlua:5]: Change the Blurb field to `Fixes some particles not displaying in multiplayer.`

File: Restore Multiplayer Particles.mlua [hunk 4]
```diff
@@ -0,0 +1,15 @@
+ --[Patch]--
+Title("Restore Multiplayer Particles")
+Author("Reimous")
+Platform("Xbox 360")
+Blurb("Adds Particles back to multiplayer")
+Description("Allows particles to be displayed in multiplayer")
```
Comment (HyperBE32) [inline Restore Multiplayer Particles.mlua:6]: Remove the Description field

**Review by hyperbx — CHANGES_REQUESTED** [review]

**Pull request merged**